politeiad/backend/gitbe/gitbe.go

```diff
@@ -2836,6 +2836,18 @@ func (g *gitBackEnd) Plugin(command, payload string) (string, string, error) {
 	case cmsplugin.CmdCastVote:
 		payload, err := g.pluginCastVote(payload)
 		return cmsplugin.CmdCastVote, payload, err
+	case cmsplugin.CmdLoadVoteResults:
+		payload, err := g.pluginLoadDCCVoteResults()
+		return cmsplugin.CmdLoadVoteResults, payload, err
+	case cmsplugin.CmdDCCVotes:
+		payload, err := g.pluginDCCVotes(payload)
```

Comment:
The command name is `dccvotes`, the function name is `pluginDCCVotes`, but the corresponding plugin struct is named `VoteResults`.  This is confusing. The naming should be consistent.